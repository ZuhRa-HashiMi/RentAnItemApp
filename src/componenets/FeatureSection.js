import React from 'react';
import { Row, Col } from 'react-bootstrap';
import home from '../assets/WhatsApp Image 2023-05-16 at 3.38.19 PM.jpeg';

export default function FeatureSection() {
  return (
    <section className="about-section feature-section">
      <Row className="mt-2">
        <Col sm={6}>
          <img src={home} className="img-fluid" alt="home" />
        </Col>
        <Col sm={6} className="pt-5">
          <h4 className="about-section__left-title pt-5">
            Only best place for your rest
          </h4>
          <p className="about-section__desc">
            More than hundreds of house at your disposal. Only the best and checked with owners.
            We will find best place for you and conditions for a safe and comfortable place.
          </p>
          <p>
            <a href="#link" className="btn btn-oval">
              Learn more
            </a>
          </p>
        </Col>
      </Row>
    </section>
  );
}
