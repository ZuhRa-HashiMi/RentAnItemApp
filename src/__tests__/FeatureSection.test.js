import React from 'react';
import renderer from 'react-test-renderer';
import FeatureSection from '../componenets/FeatureSection';

it('renders Footer correctly', () => {
  const tree = renderer.create(<FeatureSection />);
  expect(tree).toMatchSnapshot();
});
