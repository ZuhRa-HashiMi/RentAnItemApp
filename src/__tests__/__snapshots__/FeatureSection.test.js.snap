// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer correctly 1`] = `
<section
  className="about-section feature-section"
>
  <div
    className="mt-2 row"
  >
    <div
      className="col-sm-6"
    >
      <img
        alt="home"
        className="img-fluid"
        src="WhatsApp Image 2023-05-16 at 3.38.19 PM.jpeg"
      />
    </div>
    <div
      className="pt-5 col-sm-6"
    >
      <h4
        className="about-section__left-title pt-5"
      >
        Only best place for your rest
      </h4>
      <p
        className="about-section__desc"
      >
        More than hundreds of house at your disposal. Only the best and checked with owners. We will find best place for you and conditions for a safe and comfortable place.
      </p>
      <p>
        <a
          className="btn btn-oval"
          href="#link"
        >
          Learn more
        </a>
      </p>
    </div>
  </div>
</section>
`;
